src/styles/qtcstyles.scss

```diff
@@ -0,0 +1,102 @@
+/* 
+ * QTCSTYLES.SCSS: stylesheet for QTC Mapping-DBQs API. 
+*/
+
+// Variables
+// Fonts
+$serif-font-stack: "Optima", "Palatino Linotype", serif;
```

Comment:
If this project is going to grow, then using the boilerplate CSS is highly recommended.